ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-nightly.yaml

```diff
@@ -214,6 +214,30 @@ tests:
     - chain: openshift-qe-data-path-tests
     workflow: rosa-aws-sts
   timeout: 5h0m0s
+- as: perfscale-rosa-hcp-3zones-ingress-perf-3nodes-periodic
+  cron: 0 7 * * 2
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      CONFIG: config/standard-classic-rosa.yml
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      HOSTED_CP: "true"
+      MP_LABELS: node-role.kubernetes.io/infra=
+      MP_MACHINE_TYPE: r5.xlarge
+      MP_REPLICAS: "1"
+      MP_TAINTS: node-role.kubernetes.io/infra=:NoSchedule
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.13"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "3"
```

Comment:
Please update to 9 Nodes per 
https://docs.google.com/document/d/1sVkbU5ttS3g_GqfEeAWDzT25asvy_jKqy0qGCJ4Lfzo/edit#heading=h.hrk57vkfzf07